### DISCUSSION TITLE
Invisible Aircraft

### DISCUSSION THREAD

**Discussion by Busta62** [flybywiresim/aircraft#5992] [Help] [answered]
I installed the flybywire-aircraft-a320-neo into my community folder using the installer. When I select the aircraft and attempt to fly, the aircraft itself is invisible. I can switch views and rotate them. I can also hear the engine noise as it changes volume going from internal to external views. I just don't have an airplane to look at.

**Comment by Benjozork** [comment, marked as answer]
Make sure you do not have any older versions of the A32NX in "Content Manager".

**Comment by Busta62** [comment]
Thank you for the advice. I did find multiple or old installs of the a32x in my content manager. Cleaned that up and now everything is working as it should be. Thanks again.
